src/pages/blog.js

```diff
@@ -1,34 +1,69 @@
 import Head from 'next/head';
 // import { Blogs } from '../components/Global'
 
-export default function Blog({ articles }) {
+export const getStaticProps = async () => {
+  const res = await fetch('https://dev.to/api/articles?username=webxdao');
+  const data = await res.json();
+
+  return {
+    props: {
+      data,
+    },
+  };
+};
+
+const blogs = ({ data }) => {
```

Comment:
```suggestion
const Blog = ({ data }) => {
```


Hi, [Pages]https://nextjs.org/docs/basic-features/pages) should be exported with Capitalized letter! :D